modules/PhyVersoBSP/phyverso_gpio/evGpio.cpp

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#include "evGpio.h"
+#include <everest/logging.hpp>
+#include <fmt/format.h>
+
+evGpio::evGpio(evConfig& _verso_config) : verso_config(_verso_config) {
+}
+
+evGpio::~evGpio() {
+    // TODO: deinit gpios?
+}
+
+bool evGpio::init_gpios() {
+    push_buttons[CONN1_PB_STOP].set_enabled(verso_config.conf.conn1_gpio_stop_button_enabled);
+    push_buttons[CONN2_PB_STOP].set_enabled(verso_config.conf.conn2_gpio_stop_button_enabled);
+
+    if (push_buttons[CONN1_PB_STOP].get_enabled()) {
+        Everest::GpioSettings settings;
+        settings.chip_name = fmt::format("gpiochip{}", verso_config.conf.conn1_gpio_stop_button_bank);
```

Comment:
This may cause issues if the gpio device name does not start with "gpiochip". There were some discussions with Phytec to use udev rules to have persistent naming on the gpio devices, and then most likely they will have a different name. I would suggest to use the full string in the config instead of just the integer number